simulariumio/tests/data/cytosim/aster_pull3D_couples_actin_solid_3_frames/aster_pull3D_couples_actin_solid_3_frames_small.json

```diff
@@ -1 +1 @@
-{"trajectoryInfo": {"version": 1, "timeStepSize": 0.05, "totalSteps": 3, "spatialUnitFactorMeters": 1e-6, "size": {"x": 200.0, "y": 200.0, "z": 200.0}, "typeMapping": {"1": {"name": "microtubule"}, "7": {"name": "motor complex"}}}, "spatialData": {"version": 1, "msgType": 1, "bundleStart": 0, "bundleSize": 3, "bundleData": [{"frameNumber": 0, "time": 0.0, "data": [1001.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 15.0, 36.93, 36.8, 16.78, 30.55, 43.87, 19.84, 24.169999999999998, 50.93, 22.900000000000002, 17.78, 57.99999999999999, 25.95, 11.4, 65.07, 29.01, 1000.0, 12.0, 7.0, -73.8, -25.2, -43.89, 0.0, 0.0, 0.0, 2.0, 0.0]}, {"frameNumber": 1, "time": 0.05, "data": [1001.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 18.0, 44.55, 33.97, 8.86, 36.63, 38.269999999999996, 4.51, 28.96, 43.28, 0.5, 21.83, 49.61, -2.52, 16.0, 57.66, -3.58, 12.85, 66.92, -1.47, 1000.0, 12.0, 7.0, -72.52, -21.9, -43.59, 0.0, 0.0, 0.0, 2.0, 0.0]}, {"frameNumber": 2, "time": 0.1, "data": [1001.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 15.0, 44.55, 33.97, 8.86, 36.63, 38.269999999999996, 4.51, 28.96, 43.28, 0.5, 21.83, 49.61, -2.52, 16.0, 57.66, -3.58, 1000.0, 12.0, 7.0, -72.52, -21.9, -43.59, 0.0, 0.0, 0.0, 2.0, 0.0]}]}, "plotData": {"version": 1, "data": []}}
\ No newline at end of file
+{"trajectoryInfo": {"version": 1, "timeStepSize": 0.05, "totalSteps": 3, "spatialUnitFactorMeters": 1e-6, "size": {"x": 200.0, "y": 200.0, "z": 200.0}, "typeMapping": {"1": {"name": "microtubule", "geometry": {"displayType": "FIBER"}}, "7": {"name": "motor complex"}}}, "spatialData": {"version": 1, "msgType": 1, "bundleStart": 0, "bundleSize": 3, "bundleData": [{"frameNumber": 0, "time": 0.0, "data": [1001.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 15.0, 36.93, 36.8, 16.78, 30.55, 43.87, 19.84, 24.169999999999998, 50.93, 22.900000000000002, 17.78, 57.99999999999999, 25.95, 11.4, 65.07, 29.01, 1000.0, 12.0, 7.0, -73.8, -25.2, -43.89, 0.0, 0.0, 0.0, 2.0, 0.0]}, {"frameNumber": 1, "time": 0.05, "data": [1001.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 18.0, 44.55, 33.97, 8.86, 36.63, 38.269999999999996, 4.51, 28.96, 43.28, 0.5, 21.83, 49.61, -2.52, 16.0, 57.66, -3.58, 12.85, 66.92, -1.47, 1000.0, 12.0, 7.0, -72.52, -21.9, -43.59, 0.0, 0.0, 0.0, 2.0, 0.0]}, {"frameNumber": 2, "time": 0.1, "data": [1001.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 15.0, 44.55, 33.97, 8.86, 36.63, 38.269999999999996, 4.51, 28.96, 43.28, 0.5, 21.83, 49.61, -2.52, 16.0, 57.66, -3.58, 1000.0, 12.0, 7.0, -72.52, -21.9, -43.59, 0.0, 0.0, 0.0, 2.0, 0.0]}]}, "plotData": {"version": 1, "data": []}}
```

Comment:
Is this supposed to be in this PR? This is in tests/data/cytosim

Reply:
Gotcha. I'm still a little confused because the geometry data structure here is in the v3 format but everything else is not v3? Is it supposed to be like that? (e.g., `trajectoryInfo.version` says `1` and `spatialUnitFactorMeters` is not a v3 field)

Reply:
Ohh! Okay I was looking at an older diff :) sorry